compiler/AST/build.cpp

```diff
@@ -1598,6 +1598,9 @@ DefExpr* buildClassDefExpr(const char*  name,
   } else if (strcmp("_shared", name) == 0) {
     ct = installInternalType(ct, dtShared);
     ts = ct->symbol;
+  } else if (strcmp("c_ptr", name) == 0) {
+    ct = installInternalType(ct, dtCPtr);
+    ts = ct->symbol;
```

Comment:
Why do we need this and the type `dtCPtr` in the production compiler?

Reply:
It was to support `convert-uast.cpp`. but I've made `convert-uast.cpp` report "unimplemented" here like the class and record conversion cases, so this is no longer necessary and I will remove it.